src/main/java/org/umssdiplo/automationv01/core/utils/LoadPage.java

```diff
@@ -2,11 +2,12 @@
 
 import org.umssdiplo.automationv01.core.customwebdriver.ManageDriver;
 import org.umssdiplo.automationv01.core.managepage.Login.Login;
+import org.umssdiplo.automationv01.core.managepage.SmarthHouse.SHLogin;
 
 public final class LoadPage {
-    public static Login loginPage() {
+    public static SHLogin loginPage() {
```

Comment:
You need to use it, please:

public static HeaderWithLogin headerPageWithLogin() {
        ManageDriver.getInstance().getWebDriver().navigate().to(PropertyAccessor.getInstance().getBaseUrl());
        return new HeaderWithLogin();
}

public static HeaderWithoutLogin headerWithoutLogin() {
    ManageDriver.getInstance().getWebDriver().navigate().to(PropertyAccessor.getInstance().getBaseUrl());
    return new HeaderWithoutLogin();
}